### PR TITLE
chore: email analyst for 2nd review on assessment page

### PR DESCRIPTION
**Pull request by RRanath**
Implements [#NDT-239](https://connectivitydivision.atlassian.net/browse/NDT-239)

### CODE REVIEW

File: app/backend/lib/emails/templates/assesmentSecondReviewChange.ts [hunk 1]
```diff
@@ -3,21 +3,33 @@ import {
...
     emailTo: [34, 71], // Temporary IDs to handle email recipients
     emailCC: [],
     tag: 'assesment-second-review-change',
-    subject: `${initiator} requested a 2nd Review for Eligibility Screening - ${ccbcNumber}`,
+    subject: `${initiator} has requested a 2nd Review for Eligibility ${type} - ${ccbcNumber}`,
     body: `
-    <h1>${initiator} requested a 2nd Review for Eligibility Screening - ${ccbcNumber}</h1>
-    <p>${initiator} requested a 2nd Review for Eligibility Screening - ${ccbcNumber}, <a href='${url}/analyst/application/${applicationId}/assessments/screening'>Click here</a> to view the Eligibility Screening in the CCBC Portal<p>
+    <h1>${initiator} requested a 2nd Review on Eligibility ${type} - ${ccbcNumber}</h1>
+    <p>${initiator} requested a 2nd Review on Eligibility ${type} - ${ccbcNumber}, <a href='${url}/analyst/application/${applicationId}/assessments/${slug}'>Click here</a> to view the Eligibility ${type} in the CCBC Portal<p>
```
Comment (AntBush) [inline app/backend/lib/emails/templates/assesmentSecondReviewChange.ts:31]: I think the "Eligibility" here isn't part of the ticket? Might need some clarification from Mars but I believe screening is the only one we include "Eligibility" here.

**Review by AntBush — APPROVED** [review]
Looks good, just a small nit that could be addressed in another ticket

File: app/backend/lib/emails/templates/assesmentSecondReviewChange.ts (hunk 1 above)
Comment (RRanath) [reply app/backend/lib/emails/templates/assesmentSecondReviewChange.ts]: @AntBush Changed to reflect word 'assessment', please give another look whenever you can

**Review by AntBush — APPROVED** [review]
🚀

**Pull request merged**